pysph/base/particle_array.pyx

```diff
@@ -152,8 +147,8 @@ cdef class ParticleArray:
         # list of output property arrays
         self.output_property_arrays = []
 
-        if get_config().use_opencl:
-            h = ocl.DeviceHelper(self)
+        if self.backend:
```

Comment:
I don't like this, we should use this: https://github.com/pypr/pysph/blob/master/pysph/cpy/array.py#L6 and perhaps promote that to a place so we can reuse it.  This way the backend can be set once and explicitly if desired instead of have to be passed everywhere?